core/dbt/cli/params.py

```diff
@@ -6,13 +6,10 @@
 from dbt.cli.resolvers import default_project_dir, default_profiles_dir
 from dbt.version import get_version_information
 
-# TODO:  The name (reflected in flags) is a correction!
-# The original name was `SEND_ANONYMOUS_USAGE_STATS` and used an env var called "DBT_SEND_ANONYMOUS_USAGE_STATS"
-# Both of which break existing naming conventions (doesn't match param flag).
-# This will need to be fixed before use in the main codebase and communicated as a change to the community!
-anonymous_usage_stats = click.option(
-    "--anonymous-usage-stats/--no-anonymous-usage-stats",
-    envvar="DBT_ANONYMOUS_USAGE_STATS",
+# TODO:  Rename this to meet naming conventions (the word "send" is redundant)
```

Comment:
👍 Tracking this in https://github.com/dbt-labs/dbt-core/issues/6903

Reply:
Yeah, it turns out renaming a symbol in some places and not others is confusing AF.  Might as well do it all at once since it's going to require changes in a medium sized pile of tests that have a manually set value for send stats (not sure why, most of them have nothing to do with tracking).
`/shrug`